data/multus/001-rbac.yaml

```diff
@@ -19,23 +19,23 @@ rules:
       - update
 ---
 kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This is not related tright ?

Reply:
the old meanifest was outdated and static. 
Now that I'm taking the manifest from the tagged version, we get all the new stuff, such as this.